### PR TITLE
media: Revert "media: Make MediaWorker immortal"

### PR DESCRIPTION
**Pull request by junmin-kim**
This reverts commit bdaccf67cbfe19a33cfbf7e438d06b4da6205c50.

### CODE REVIEW

**Comment by junmin-kim** [comment]
Please refer this commit https://github.com/Samsung/TizenRT/pull/1912/commits/b18041a0f791dbfd049e96d601e43e8803416c5f

**Comment by seinfra** [comment]
Target : [be2e444eab0e4d620af85d6d640b9a12d0e5d100] - Code Rule Check Result:
framework/src/media/MediaWorker.h:37: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:WxV)
framework/src/media/MediaWorker.h:37: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxE)
framework/src/media/MediaWorker.h:38: ERROR: [BRC_M_FTN] open brace '{' following function declarations go on the next line
framework/src/media/MediaWorker.h:38: ERROR: [SPC_M_OPR] spaces required around that '&&' (ctx:WxV)
framework/src/media/MediaWorker.h:38: ERROR: [SPC_M_OPR] spaces required around that '&&' (ctx:WxV)
framework/src/media/MediaWorker.h:41: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaWorker.h:41: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaWorker.h:41: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/src/media/MediaWorker.h:53: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaWorker.h:53: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaWorker.h:53: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/src/media/MediaWorker.h:56: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)

File: framework/src/media/MediaWorker.cpp [hunk 1]
```diff
@@ -37,22 +36,19 @@ void MediaWorker::startWorker()
...
-				return;
-			}
-			pthread_setname_np(mWorkerThread, mThreadName);
+		int ret;
+		pthread_attr_t attr;
+		pthread_attr_init(&attr);
+		pthread_attr_setstacksize(&attr, mStacksize);
+		mIsRunning = true;
+		ret =
+			pthread_create(&mWorkerThread, &attr, static_cast<pthread_startroutine_t>(MediaWorker::mediaLooper), this);
```
Comment (HONGCHAEHEE) [inline framework/src/media/MediaWorker.cpp]: Put it in the previous line.

Comment (junmin-kim) [reply framework/src/media/MediaWorker.cpp]: Fixed it

**Comment by seinfra** [comment]
Target : [08fffd18a19527f0f29c696b7f99ee02446e0dea] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [08fffd18a19527f0f29c696b7f99ee02446e0dea] - Code Rule Check Result:
framework/src/media/MediaWorker.h:37: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:WxV)
framework/src/media/MediaWorker.h:37: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxE)
framework/src/media/MediaWorker.h:38: ERROR: [BRC_M_FTN] open brace '{' following function declarations go on the next line
framework/src/media/MediaWorker.h:38: ERROR: [SPC_M_OPR] spaces required around that '&&' (ctx:WxV)
framework/src/media/MediaWorker.h:38: ERROR: [SPC_M_OPR] spaces required around that '&&' (ctx:WxV)
framework/src/media/MediaWorker.h:41: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaWorker.h:41: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaWorker.h:41: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/src/media/MediaWorker.h:53: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaWorker.h:53: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaWorker.h:53: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/src/media/MediaWorker.h:56: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)

**Pull request merged**